<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualisierung 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1001</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No data loaded</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>657</x>
      <y>20</y>
      <width>521</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_MIP">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MIP</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_AlphaCompositing">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>110</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Compositing</string>
    </property>
   </widget>
   <widget class="QSlider" name="sl_AlphaCompositing">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>140</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Comp Treshold</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>60</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Number of Samples per Pixel, </string>
     </property>
     <property name="toolTipDuration">
      <number>5</number>
     </property>
     <property name="text">
      <string>Sample size</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_sampleSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Number of Samples per Pixel, </string>
     </property>
     <property name="toolTipDuration">
      <number>5</number>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;20&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_transform">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>270</y>
      <width>171</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Transformations</string>
    </property>
    <widget class="QSlider" name="sl_xTranslate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X-Translate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y-Translate</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_yTranslate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_rotateX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate around x-Axis [deg]</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_rotateY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate around y-Axis [deg]</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_yRotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>45</number>
     </property>
    </widget>
    <widget class="QSlider" name="sl_xRotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>45</number>
     </property>
    </widget>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>59</y>
      <width>991</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>991</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bu_render">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>630</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
   </widget>
   <zorder>openGLWidget</zorder>
   <zorder>graphicsView</zorder>
   <zorder>groupBox</zorder>
   <zorder>labelTop</zorder>
   <zorder>progressBar</zorder>
   <zorder>cb_MIP</zorder>
   <zorder>cb_AlphaCompositing</zorder>
   <zorder>sl_AlphaCompositing</zorder>
   <zorder>label_2</zorder>
   <zorder>gb_transform</zorder>
   <zorder>bu_render</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
